src/styles/stars.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+  align-items: baseline;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), using `align-items: baseline;` is not typical for a flex container that arranges icons. If you want all stars to be aligned in a straight line, it's better to use `align-items: center;`. This will vertically center the stars within the container.